pkg/api/types.go

```diff
@@ -545,6 +559,10 @@ type PersistentVolumeClaimSpec struct {
 	// More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1
 	// +optional
 	StorageClassName *string
+	// volumeMode defines what type of volume is required by the claim.
+	// values include Filesystem or Block and default is Filesystem if not present
+	// +optional
+	VolumeMode *PersistentVolumeMode
```

Comment:
There should be a note that this field is alpha.
There should be a note that "Filesystem" is implied when no mode is specified (word "default" is not appropriate here as there is no defaulting..."